core/trie/node.go

```diff
@@ -26,7 +26,6 @@ func (n *Node) Hash(path *Key, hashFunc hashFunc) *felt.Felt {
 	}
 
 	pathFelt := path.Felt()
-	// https://docs.starknet.io/documentation/develop/State/starknet-state/
```

Comment:
I couldn't find a better link and this one doesn't make sense

Reply:
I believe it should be this one - https://docs.starknet.io/architecture-and-concepts/network-architecture/starknet-state/#trie_construction

Reply:
done :)